apps/state/lib/state/routes_by_service.ex

```diff
@@ -104,6 +104,9 @@ defmodule State.RoutesByService do
       end)
       |> Enum.map(fn {x, y} -> {x, Enum.map(y, fn x -> x.route_id end)} end)
 
+    #IO.puts("items")
+    #IO.inspect(items)
```

Comment:
These should be deleted. Also a tip, you can use `IO.inspect(items, label: "items")` to let Elixir annotate the output.